common/index.ts

```diff
@@ -0,0 +1,4 @@
+export const PLUGIN_ID = 'opendistroKibanaReports';
+export const PLUGIN_NAME = 'opendistro_kibana_reports';
```

Comment:
any reason its changed from `opendistro-kibana-reports`?

Reply:
Not any big reasons, just that I saw we use `_` more often than `-` in the codebase, so I think it's better we keep the same style